providers/mixpanel.go

```diff
@@ -0,0 +1,30 @@
+package providers
+
+const Mixpanel Provider = "mixpanel"
+
+func init() {
+	// Mixpanel configuration
+	// workspace maps to either region(api, api-eu) or server(data,data-eu).
+	// Supported Mixpanel APIs
+	// -	Ingestion API
+	// -	Identity API
+	// -	Event Export API
+	// -	Data Pipelines API
+	SetInfo(Mixpanel, ProviderInfo{
+		AuthType: Basic,
+		BaseURL:  "https://{{.workspace}}.mixpanel.com",
```

Comment:
workspace is meant for user or project specific subdomains.

But actual variables are just API subdomains. Please choose another term instead of ``{{.workspace}}` maybe something like `{{.apisubdomain}}`or `{{.apiservicesubdomain}}`or something else you find more appropriate.